shell/src/main/java/tachyon/shell/command/LoadCommand.java

```diff
@@ -63,25 +61,22 @@ void runCommand(TachyonURI path) throws IOException {
    */
   private void load(TachyonURI filePath) throws IOException {
     try {
-      TachyonFile fd = mTfs.open(filePath);
-      FileInfo fInfo = mTfs.getInfo(fd);
-      if (fInfo.isFolder) {
-        List<FileInfo> files = mTfs.listStatus(fd);
-        Collections.sort(files);
-        for (FileInfo file : files) {
-          TachyonURI newPath = new TachyonURI(file.getPath());
+      URIStatus status = mTfs.getStatus(filePath);
+      if (status.isFolder()) {
+        List<URIStatus> statuses = mTfs.listStatus(filePath);
+        for (URIStatus uriStatus : statuses) {
+          TachyonURI newPath = new TachyonURI(uriStatus.getPath());
```

Comment:
I'm not sure what the point of sorting was, since `FileInfo`'s comparison isn't really useful.